apps/src/code-studio/pd/application/teacher1920/Section2ChooseYourProgram.jsx

```diff
@@ -26,6 +26,36 @@ export default class Section2ChooseYourProgram extends LabeledFormComponent {
   }
 
   render() {
+    // This should be kept consistent with the calculation logic in
+    // dashboard/app/models/pd/application/teacher1920_application.rb.
```

Comment:
:+1: / :sob:

Reply:
In an ideal world, it seems it'd be nice if we drove these forms more using ajax.. then we could have this code on the server but call it repeatedly from the client.